### PR TITLE
Update projects to use latest WebView2 SDK 1.0.2470-prerelease

### PR DESCRIPTION
**Pull request by oggy22**
* Updates for Win32, WPF, WinForms, UWP and WinUI3 sample apps from 123.0.2470.0
* Updated package version for Win32, WPF and WinForms sample apps to 1.0.2470-prerelease

### CODE REVIEW

File: SampleApps/WebView2APISample/ScenarioFileSystemHandleShare.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                wil::com_ptr<ICoreWebView2ExperimentalEnvironment14> environment_staging14 =
+                    environment.try_query<ICoreWebView2ExperimentalEnvironment14>();
+                CHECK_FEATURE_RETURN_HRESULT(environment_staging14);
+                wil::com_ptr<ICoreWebView2ExperimentalFileSystemHandle> rootHandle;
+                CHECK_FAILURE(environment_staging14->CreateWebFileSystemDirectoryHandle(
+                    L"C:\\", COREWEBVIEW2_FILE_SYSTEM_HANDLE_PERMISSION_READ_ONLY,
+                    &rootHandle));
+                wil::com_ptr<ICoreWebView2ExperimentalObjectCollection> webObjectCollection;
+                IUnknown* webObjects[] = {rootHandle.get()};
+                CHECK_FAILURE(environment_staging14->CreateObjectCollection(
```
Comment (lflores-ms) [inline SampleApps/WebView2APISample/ScenarioFileSystemHandleShare.cpp:46]: Is this meant to be here?

Comment (lflores-ms) [reply SampleApps/WebView2APISample/ScenarioFileSystemHandleShare.cpp:46]: naming is off

**Review by lflores-ms — APPROVED** [review]
Naming-only issue, shouldn't block SDK.

File: SampleApps/WebView2APISample/ScenarioFileSystemHandleShare.cpp (hunk 1 above)
Comment (peiche-jessica) [reply SampleApps/WebView2APISample/ScenarioFileSystemHandleShare.cpp:46]: /cc @yildirimcagri-msft

**Pull request merged**